packages/ui-components/src/components/text-field/text-field.base.scss

```diff
@@ -1,5 +1,7 @@
 @import '../../assets/styles/globals';
 
+$right-slot-width: 36px;
```

Comment:
This variable should be customizable, please move it to the `:host` selector as a CSS property with this value as default.

Reply:
Setted as default variable